TRIAGING.md

```diff
@@ -12,19 +12,29 @@ You can track if your issue was triaged by watching your GitHub notifications fo
 
 Each meeting we seek to address all new issues. However, should we run out of time before your issue is discussed, you are always welcome to attend the next meeting or to follow up on the issue post itself.
 
-### How do I join the Triage meeting?
+### How many triage meetings are there?
 
-Meetings are hosted regularly at 10:00a - 10:55a Central Time every Wednesday and can be joined via [Zoom](https://zoom.us/download), with this [meeting link](https://us02web.zoom.us/j/86287450465) and passcode `805212`.
+The OpenSearch repository is very large and as a result, the rate of incoming issues and technical expertise required is too large for any one person to be able to effectively address all issues. Therefore, we have multiple triage meetings scoped to the following component areas: Search, Storage, Cluster Manager, and finally "Core" as a catch-all for all other issues. Component labels are used to filter issues, and each triage meeting will use these labels to construct a search as follows to target issues:
 
-After joining the Zoom meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam or microphone available, you can still join in via the text chat.
+- [Search](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged+label%3A%22Search%22%2C%22Search%3ARemote+Search%22%2C%22Search%3AResiliency%22%2C%22Search%3APerformance%22%2C%22Search%3ARelevance%22%2C%22Search%3AAggregations%22%2C%22Search%3AQuery+Capabilities%22%2C%22Search%3AQuery+Insights%22%2C%22Search%3ASearchable+Snapshots%22%2C%22Search%3AUser+Behavior+Insights%22)
```

Comment:
What do you think about moving these search urls, the existing agenda doesn't take these into account on line 41 (bullet 4).  I'd rather these were build into the steps that are followed, otherwise the person running triage needs to remember and reference up in the document.

Could we bake these into that link so the person running triage only needs to click a link according the meeting they are in?

Reply:
Sure, I'll do that